### PR TITLE
Get exception traceback in suite easily

### LINKED ISSUE #1169: [FEAT] Attach exception traces to suite result.
**Is your feature request related to a problem? Please describe.**
Currently it's often hard to debug exceptions in checks running as parts of suites - all you can see is the error message. 

**Describe the solution you'd like**
Attach the exception trace as part of the result object.

### PR DESCRIPTION
**Pull request by benisraeldan**
#### Reference Issues/PRs

Resolves #1169

#### What does this implement/fix? Explain your changes.
add `get_errors()` func to suite results for debugging. and print help message in checks that weren't displayed.
Screen Shot 2022-04-13 at 17 11 58

### CODE REVIEW

**Review by matanper — COMMENTED** [review]
I don't understand what "contains_check_failures" is used for
also what is the debug message used for

**Comment by matanper** [comment]
> I don't understand what "contains_check_failures" is used for also what is the debug message used for

NVM, got it but seems too complicated. Why not just add the message as static in the failure tabs?

File: deepchecks/core/display_suite.py [hunk 1]
```diff
@@ -168,7 +169,8 @@ def _display_suite_widgets(summary: str,
         with warnings.catch_warnings():
             warnings.simplefilter(action='ignore', category=FutureWarning)
             others_df = dataframe_to_html(others_table.style.hide_index())
-        h2_widget = widgets.HTML(_CHECKS_WITHOUT_DISPLAY_TITLE)
+        debug_message = '</br>For call traces use get_errors() on result object</br>' if contains_check_failures else ''
```
Comment (matanper) [inline deepchecks/core/display_suite.py:172]: Phrasing is a bit weird "For call traces"

**Comment by benisraeldan** [comment]
> Why not just add the message as static in the failure tabs?
the `Other Checks that wern't Displayed` section also contains check without output and without condictions "nothing found checks" so it would be weird telling the user how to debug it when there is nothing to debug

**Review by JKL98ISR — APPROVED** [review]

**Pull request merged**